app/views/codequest/Header.vue

```diff
@@ -0,0 +1,341 @@
+        { src: '/images/pages/codequest/coding/coding_2.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_3.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_4.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_5.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_6.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_7.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_8.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_9.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_10.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_11.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_12.jpg', alt: 'Header image' },
+        { src: '/images/pages/codequest/coding/coding_13.jpg', alt: 'Header image' }
+      ]
+    }
+  },
+  created () {
+    setTimeout(() => {
+      this.currentImage = 0
+    }, 0)
+    setInterval(() => {
+      this.currentImage = (this.currentImage + 1) % this.images.length
+    }, 3000)
+  }
+}
+</script>
+
+<style scoped lang="scss">
+.header {
+    background-color: #ffffff;
+    position: relative;
+    padding-top: 170px;
+
+    .overlap-group {
+        position: relative;
+    }
+
+    .content {
+        align-items: flex-start;
+        display: flex;
+        gap: 30px;
+
+        @media (max-width: 991px) {
+            flex-direction: column;
+            align-items: center;
+            gap: 60px;
+        }
+    }
+
+    .header-text {
+        align-items: flex-start;
+        display: flex;
+        flex-direction: column;
+        gap: 40px;
+        justify-content: center;
+        position: relative;
+        flex: 0 0 calc(40% - 15px);
+        z-index: 2;
+    }
+
+    .header-image {
+        flex: 0 0 calc(60% - 15px);
+        position: relative;
+        aspect-ratio: 5472/3648;
+        width: 100%;
+
+        .cubes {
+            position: absolute;
+            z-index: 1;
+            width: 140%;
+            top: -60%;
+            left: -22%;
+            transform: rotate(12deg);
+        }
+
+        .image {
+            z-index: 2;
+            width: 100%;
+            box-shadow: 0px 6px 22px 0px rgba(0, 0, 0, 0.10);
+            border-radius: 24px;
+            position: absolute;
+            aspect-ratio: 5472 / 3648;
+            overflow: hidden;
+
+            transition: opacity 1s ease-in-out;
+            opacity: 0;
+
+            &.is-visible {
+                opacity: 1;
+            }
+
+            img {
+                position: absolute;
+                top: 0;
+                left: 0;
+                width: 100%;
+                height: 100%;
+                object-fit: cover;
+            }
+        }
+
+        @media (max-width: 991px) {
```

Comment:
we have screen-md-min and other variables setup. It's better to use them instead of magic-numbers